<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortefeuilleInput</class>
 <widget class="QDialog" name="PortefeuilleInput">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modifier infos portefeuille</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="topMargin">
      <number>30</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="isinLabel">
       <property name="text">
        <string>&amp;Isin portefeuille</string>
       </property>
       <property name="buddy">
        <cstring>isinLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="isinLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="nomDuPortefeuilleLabel">
       <property name="text">
        <string>&amp;Nom du portefeuille</string>
       </property>
       <property name="buddy">
        <cstring>nomDuPortefeuilleLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nomDuPortefeuilleLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PortefeuilleInput</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PortefeuilleInput</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
